cmd/jujud/agent/machine/manifolds.go

```diff
@@ -703,6 +707,41 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 			StateName: stateName,
 			NewWorker: auditconfigupdater.New,
 		})),
+
+		raftTransportName: ifController(rafttransport.Manifold(rafttransport.ManifoldConfig{
+			ClockName:         clockName,
+			AgentName:         agentName,
+			AuthenticatorName: httpServerName,
+			HubName:           centralHubName,
+			MuxName:           httpServerName,
+			DialConn:          rafttransport.DialConn,
+			NewWorker:         rafttransport.NewWorker,
+			Path:              "/raft",
+		})),
+
+		// TODO(babbageclunk): not sure what gates should be around this.
```

Comment:
As long as the raft transport is a dependency, you shouldn't need other gates.

Reply:
Ok, I'll remove this.